src/Model/AliasesDrushRcEmitter.php

```diff
@@ -34,6 +34,20 @@ public function write(AliasCollection $collection)
         $drushConfigFiltered = implode("\n", array_filter($drushConfig, array($this, 'filterForPantheon')));
         $drushConfigFiltered .= "\n" . '$options["include"][] = drush_server_home() . "/.drush/pantheon/drush8";';
         $drushRCEditor->writeDrushConfig($drushConfigFiltered);
+
+        //copy policy docs 
+
+        $policyFromPath = 'policy/drush8/pantheon_policy.drush.inc';
+        $policyToPath = $this->base_dir . "/pantheon/drush8";
+        if (is_dir($policyToPath)) {
+            rmdir($policyToPath . '/pantheon_policy.drush.inc');
+        }
+        if (!file_exists($policyToPath)) {
+            mkdir($policyToPath, 0777, true);
```

Comment:
mkdir fails if any of the parent directories do not exist. symfony/filesystem has a `mkdir` method that works recursively.

Reply:
yeah I had that occur so i looked it up and apparently mkdir has a "recursive" feature (which is why i put true as my third arg), but if you think its better to use the symfony/filesystem mkdir i can change it to that!

Reply:
Yeah I didn't notice the third parameter because old PHPs (that we still need to support) don't have it, so it's invisible to me. :p